rpcs3/Emu/Io/GunCon3.cpp

```diff
@@ -220,6 +220,13 @@ void usb_device_guncon3::interrupt_transfer(u32 buf_size, u8* buf, u32 endpoint,
 		return;
 	}
 
+	if (m_controller_index >= g_cfg_guncon3.players.size())
```

Comment:
just ensure in the constructor. This should never happen.

Reply:
guncon controllers are selectable for every player from 1 to 7, it would just crash the game